spec/cocoapods/generate/configuration_spec.rb

```diff
@@ -186,6 +224,34 @@ def should_raise(*args, &blk)
     end
   end
 
+  describe_method :gen_dir_for_specs do
```

Comment:
new tests added here that were missing before but also test the new logic added.